test/multiverse/suites/active_record/db/migrate/20150414084400_create_groups.rb

```diff
@@ -3,7 +3,9 @@
 # See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
 # frozen_string_literal: true
 
-class CreateGroups < ActiveRecord::VERSION::STRING >= "5.0.0" ? ActiveRecord::Migration[5.0] : ActiveRecord::Migration
```

Comment:
All `active_record/db/migrate` test files had this line, which was added 5 years ago when ActiveRecord 5.0+ started requiring the version. Rather than pin the version to 5.0, I've included it in this update to match the dynamic versioning we're adding elsewhere.

For comparison, the new line/method call reads:
ActiveRecord::VERSION::STRING >= "5.0.0" ? ActiveRecord::Migration["#{ActiveRecord::VERSION::STRING[0]}.0"] : ActiveRecord::Migration